steps/UpdateSkill.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using TechTalk.SpecFlow;
+
+namespace MarsQA_1.Feature
+{
+    [Binding]
+    public class UpdateSkillSteps
+    {
+       
+        [Given(@"I click update and update skill and level")]
+        public void GivenIClickUpdateAndUpdateSkillAndLevel()
+        {
+            Pages.Skills.Update_Skill();

```

Comment:
Again, please instantiate that class and call it, instead of adding the whole class here